### ISSUE TITLE
apwp.py and plot_poles_colorbar example 

### ISSUE THREAD

**Issue by lfairchild** [PmagPy/2016_Tauxe-et-al_PmagPy_Notebooks#1]
In the Examples notebook, I'm currently trying to demonstrate pulling out poles from the built-in 'apwp' function and plotting them using the plot_pole_colorbar function. I had to work with the formatting of the apwp output a little to get it into a pandas dataframe (maybe this is something that should eventually be implemented in the actual code), but, once I did this, the plotting function really does not like the values for some reason, and I can't figure out why. I feel like I've tried tweaking everything and comparing to the working APWP plot in the Slate Islands notebook, but nothing has worked so far. I feel like there's something obvious I'm missing with the data format, as the plot_pole_colorbar function is working fine with the Torsvik (2012) data in the Slate Islands notebook. (Another option would be to copy the APWP plot and Torsvik data from the Slate Islands notebook, but I figured I'd try to demonstrate the built-in apwp() function while I was at it.)

**Comment by Swanson-Hysell** [comment]
Are all of the numbers in the apwp.py output floats? That could be the
issue.

---

Nicholas Swanson-Hysell
Assistant Professor
Department of Earth and Planetary Science
University of California, Berkeley

mobile number: 510.542.4787
http://www.swanson-hysell.org

**Comment by lfairchild** [comment]
I've added a simplified example of plotting VGPs to the notebook. I still thought it might be nice to also demonstrate plot_pole_colorbar(), so in lieu of apwp.py I just added the Torsvik (2012) APWP data to a folder in the main repo. I think it could be a nice demonstration, but I'm open to scrapping it too if it doesn't seem necessary. The figure (accompanied by some text) is in the notebook now.
